cmd/scheduler/options/options.go

```diff
@@ -83,6 +84,7 @@ func New(origArgs []string) *Options {
 	fs.StringSliceVar(&opts.EtcdInitialPeers, "initial-cluster", []string{"dapr-scheduler-server-0=http://localhost:2380"}, "Initial etcd cluster peers")
 	fs.StringVar(&opts.EtcdDataDir, "etcd-data-dir", "./data", "Directory to store scheduler etcd data")
 	fs.StringSliceVar(&opts.EtcdClientPorts, "etcd-client-ports", []string{"dapr-scheduler-server-0=2379"}, "Ports for etcd client communication")
```

Comment:
```suggestion
	fs.StringSliceVar(&opts.EtcdClientPorts, "etcd-client-grpc-ports", []string{"dapr-scheduler-server-0=2379"}, "Ports for etcd client grpc communication")
```
if we are being explicit on http, it might be worth putting grpc here

Reply:
This is related to decision we're gonna make above (re: to having a default http port or not). If we do it like etcd does it and serve http on the same port as grpc by default, it wouldn't make sense to add `grpc` to the parameter.